hot-desk/src/main/resources/db.changelog/changes/employee.yaml

```diff
@@ -66,3 +66,21 @@ databaseChangeLog:
                   name: em_modified
                   type: datetime
                   defaultValueComputed: CURRENT_TIMESTAMP
+  - changeSet:
+      id: add-columns
+      author: saidaminqambarov408@gmail.com
```

Comment:
Please create separate file for this changeset

Reply:
Done